app/components/gh-post-settings-menu/email.js

```diff
@@ -88,11 +88,7 @@ export default Component.extend({
 
     // TODO: put this on settings model
     _isMailgunConfigured() {
```

Comment:
I think this can be removed entirely now? The equivalent of `get(settings, 'bulkEmailSettings.isEnabled')` can be used anywhere that `_isMailgunConfigured` is used